doc/guides/backporting-to-release-lines.md

```diff
@@ -6,54 +6,58 @@ Each release line has a staging branch that the releaser will use as a scratch
 pad while preparing a release. The branch name is formatted as follows:
 `vN.x-staging` where `N` is the major release number.
 
-*Note*: For the active staging branches see the [Release Schedule][].
+### Active staging branches
+
+| Release Line | Staging Branch |
+| ------------ | -------------- |
+| `v7.x`       | `v7.x-staging` |
+| `v6.x`       | `v6.x-staging` |
+| `v4.x`       | `v4.x-staging` |
```

Comment:
Are all these changes intentional? I think your commit might have been based on an old copy of master.

Reply:
@gibfahn You are right. Sorry for that. Please review.